feature-libs/cart/quick-order/components/quick-order/quick-order.component.spec.ts

```diff
@@ -47,6 +48,14 @@ class MockGlobalMessageService implements Partial<GlobalMessageService> {
   ): void {}
 }
 
+const mockData: any = {
```

Comment:
You can add the proper type with that newly created interface

`CmsQuickOrderComponent`